manageiq-messaging.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'activesupport', '~> 5.2.4.3'
   spec.add_dependency 'ruby-kafka', '~> 1.0'
+  spec.add_dependency 'rdkafka', '~> 0.8'
```

Comment:
If there are multiple optional dependencies, then we shouldn't list them here, or perhaps we should only list as development dependencies.  Then the caller can specify the gem they want.

This is similar to how activerecord doesn't require the mysql gem of your are using pg.

We may have to do this as a breaking change however for denver